flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/operations/SqlRTASNodeToOperationConverterTest.java

```diff
@@ -61,6 +62,27 @@ public void testReplaceTableAs() {
         testCommonReplaceTableAs(sql, tableName, tableComment);
     }
 
+    @Test
+    public void testExplainCreateOrReplaceTableAs() {
```

Comment:
I don't think we need this test. The explain test below that outputs the full pipeline should be sufficient.

Reply:
Done